sdk/purview/azure-purview-workflow/azure/purview/workflow/_vendor.py

```diff
@@ -17,18 +17,6 @@
     from ._serialization import Deserializer, Serializer
 
 
-def _format_url_section(template, **kwargs):
```

Comment:
Deleted method for the pylint error I received because this method is unused. This was the error:
`azure/purview/workflow/_vendor.py:20: [R1710(inconsistent-return-statements), _format_url_section] Either all return statements in a function should return an expression, or none of them should.`